web/forms.py

```
@@ -1945,3 +1949,46 @@
         if len(title) < 5:
             raise forms.ValidationError(_("Title too short"), code="invalid_length", params={"min_length": 5})
         return title
+
+
+class VirtualClassroomForm(forms.ModelForm):
+    """Form for creating and editing virtual classrooms."""
+    def __init__(self, *args, **kwargs):
```

Ruff (0.11.9)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)